src/cpp/redis.cpp

```diff
@@ -142,9 +120,8 @@ CommandReply Redis::put_tensor(TensorBase& tensor)
     return this->run(cmd);
 }
 
-CommandReply Redis::get_tensor(const std::string& key)
+CommandReply Redis::get_tensor(const std::string& key, GetTensorCommand& cmd)
```

Comment:
Same as before, we probably don't need to pass through the ``GetTensorCommand``.